### DISCUSSION TITLE
Is there documentation for bootstrap command (npx wxt@latest init)?

### DISCUSSION THREAD

**Discussion by reuse-ay** [wxt-dev/wxt#822] [Q&A] [answered]
I'm wondering if  [command which bootstraps project](https://wxt.dev/get-started/installation.html#bootstrap-project) has options such as directory specification.

**Comment by aklinker1** [comment, marked as answer]
Just the help docs: https://wxt.dev/api/cli/wxt-init.html

This is the same text you see when running `wxt init --help`

**Reply by reuse-ay** [reply]
Thank you!
